wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container fluid fill-height justify-center>
+    <v-card width=80%>
+      <v-card-title>Leaderboard</v-card-title>
```

Comment:
You may consider using a data table in the future just to make the display of data and formatting easier to control:
https://vuetifyjs.com/en/components/data-tables/